<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopHome</class>
 <widget class="QWidget" name="DesktopHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Integração de Colaboradores REP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
	background-image: url(:/img/fundoBG.png);
}

QGroupBox {
	background-image: url(:/img/fundoBG.png);
}


/* Nice Windows-XP-style password character. */
QLineEdit[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}

/* We provide a min-width and min-height for push buttons
   so that they look elegant regardless of the width of the text. */
QPushButton {
    background-color: #FFFFFF;
    border-width: 0px;
    border-color: #FF3300;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: #66CCFF;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #66FFFF;
}

QLabel, QAbstractButton {
    font: bold;
}

/* Mark mandatory fields with a brownish color. */
.mandatory {
    color: #66CCFF;
}

/* Custumizando combobox*/
QComboBox {
}
/* Bold text on status bar looks awful. */
QStatusBar QLabel {
   font: normal;
}

QStatusBar::item {
    border-width: 0;
    border-color: #FF3300;
    border-style: solid;
    border-radius: 1;
}

QComboBox, QLineEdit, QSpinBox, QTextEdit, QListView {
    background-color: #FFFFFF;
    selection-color: #003399; 
    selection-background-color: #66FFFF;
}

QListView {
    show-decoration-selected: 1;
}

QListView::item:hover {
    background-color: #66FFFF;
}

/* We reserve 1 pixel space in padding. When we get the focus,
   we kill the padding and enlarge the border. This makes the items
   glow. */
QLineEdit, QFrame, QComboBox, SpinBox, CheckBox {
    border-width: 0px;
    padding: 1px;
    border-style: solid;
    border-color: #FF3300;
    border-radius: 2px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus, QFrame:focus {
    border-width: 0.5px;
    padding: 0px;
}

/* A QLabel is a QFrame ... */
QLabel {
    border: none;
    padding: 0;
    background: none;
}

/* A QToolTip is a QLabel ... */
QToolTip {
    border: 1px solid #FF3300;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

/* Nice to have the background color change when hovered. */
QRadioButton:hover, QCheckBox:hover {
    background-color: #99CCFF;
}

/* Force the dialog's buttons to follow the Windows guidelines. */
QDialogButtonBox {
    button-layout: 0;
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutDesktopHome">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grupoPrincipal">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="8" column="4" colspan="2">
         <widget class="QPushButton" name="botaoCancelar">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="botaoIntegrar">
          <property name="text">
           <string>Integrar [REP]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="3">
         <widget class="QDateEdit" name="dataFinal">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2020</year>
            <month>8</month>
            <day>22</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Filial :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Admissao :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Colaborador :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QPushButton" name="botaoPesquisaFilial">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QPushButton" name="botaoAplicarData">
          <property name="text">
           <string>Hoje</string>
          </property>
         </widget>
        </item>
        <item row="8" column="6" colspan="2">
         <widget class="QPushButton" name="botaoSair">
          <property name="text">
           <string>Sair</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="6">
         <widget class="QLineEdit" name="campoCodigoFilial">
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="2">
         <widget class="QPushButton" name="botaoDemitir">
          <property name="text">
           <string>Demitir [REP]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="6">
         <widget class="QLineEdit" name="campoCodigoColaborador"/>
        </item>
        <item row="4" column="7">
         <widget class="QPushButton" name="botaoPesquisaColaborador">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="botaoVisulizar">
          <property name="text">
           <string>Visualizar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="8">
         <widget class="QComboBox" name="comboBoxEmpresas"/>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDateEdit" name="dataInicial">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2020</year>
            <month>8</month>
            <day>22</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="8">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Progresso :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="7">
         <widget class="QProgressBar" name="barraProgresso">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dataInicial</tabstop>
  <tabstop>dataFinal</tabstop>
  <tabstop>campoCodigoFilial</tabstop>
  <tabstop>campoCodigoColaborador</tabstop>
  <tabstop>botaoVisulizar</tabstop>
  <tabstop>botaoIntegrar</tabstop>
  <tabstop>botaoDemitir</tabstop>
  <tabstop>botaoCancelar</tabstop>
  <tabstop>botaoSair</tabstop>
  <tabstop>comboBoxEmpresas</tabstop>
  <tabstop>botaoAplicarData</tabstop>
  <tabstop>botaoPesquisaFilial</tabstop>
  <tabstop>botaoPesquisaColaborador</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
